examples/preact-kitchen-sink/src/stories/index.stories.js

```diff
@@ -0,0 +1,21 @@
+import { h } from 'preact';
+
+import { storiesOf } from '@storybook/preact';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+
+import Button from '../Button';
+
+storiesOf('Welcome', module).add('to Storybook', () => (
+  <Button onclick={linkTo('Button')}>This button links to Button</Button>
```

Comment:
The welcome story is a `<Button>` component 🤣 ?

Reply:
Woops thanks, was still missing that one!